.landscaper/deploy_items.yaml

```
@@ -0,0 +1,62 @@
+- name: registration
+  type: Helm
+  config:
+    apiVersion: helm.deployer.landscaper.gardener.cloud/v1alpha1
+    kind: ProviderConfiguration
+    kubeconfig: {{ .imports.cluster.spec.config.kubeconfig | b64enc }}
```

```suggestion
  type: Helm
  target:
    name: {{ .imports.cluster.metadata.name }}
    namespace: {{ .imports.cluster.metadata.namespace }}
  config:
    apiVersion: helm.deployer.landscaper.gardener.cloud/v1alpha1
    kind: ProviderConfiguration
```

`config.kubeconfig` is also possible but specifying the target fits better into the landscaper concepts.